Tools/parametric_model/configs/vtol_tiltrotor_longitudinal_model.yaml

```diff
@@ -14,7 +19,7 @@ model_config:
     rotors:
       # All rotors in the same group will share the coefficients
       puller_:
-        - rotor_4:
+        - rotor:
```

Comment:
So this means that the rotor doesn't really tilt...right?

Reply:
Yes, the current implementation assumes this and the only difference to the fixed-wing model are the modified topics according to the new conventions. I took this simple approach, as the model does not work for anything else than fixed-wing flight anyways.